src/Dashboard/Dashboard.js

```diff
@@ -23,35 +24,40 @@ const Dashboard = (props) => {
     scenarioName,
     scenarioId,
     scenarioState,
+    noScenario,
     ...otherProps
   } = props;
   const formattedUrl = url.replaceAll('<ScenarioName>', scenarioName).replaceAll('<ScenarioId>', scenarioId);
+  const { t } = useTranslation();
 
   // Handle optional status property
   const noRun = scenarioState === 'Created' || scenarioState === null;
   const runInProgress = scenarioState === 'Running';
   const hasError = scenarioState === 'Failed';
-  const isReady = scenarioState === undefined || scenarioState === 'Successful';
+  const isReady = (scenarioState === undefined || scenarioState === 'Successful') && noScenario === false;
```

Comment:
`noScenario` is not marked as `required`, so it can be `undefinied`, which is not handled here.
Either mark it required, or handle the `undefined` case.

Reply:
Another solution, the initial chosen one : add a default value to noScenario props (false).

Reply:
could be replaced by `!noScenario` instead of  `noScenario === false `

Reply:
Done